app.js

```diff
@@ -0,0 +1,23 @@
+const body = document.querySelector("body");
+const initialContent = document.querySelector(".initial-content");
+const mainKeycode = document.querySelector(".keycode-info");
+const keyCard = document.querySelector(".key-card");
+const codeCard = document.querySelector(".code-card");
+const whichCard = document.querySelector(".which-card");
```

Comment:
If these are intended to be unique (and it looks like you are), consider using id="keycard" and "#keycard" rather than classes :)

Reply:
Ah yes! Thank you!